firefox-ios/Client/Frontend/Home/Homepage Rebuild/Merino/MerinoMiddleware.swift

```diff
@@ -44,7 +45,7 @@ final class MerinoMiddleware {
     private func getHomepageStoriesAndUpdateState(for action: Action) {
         Task {
```

Comment:
So since we are under that `@MainActor` middleware, that `Task` is implicitly `@MainActor`?